app/sdk/converter/ObjectConverter.java

```diff
@@ -847,20 +847,26 @@ public static void copyFromAttachment(DataSetItemAttachment attachmentItem,
                                            boolean useGetterAndSetter, boolean value,
                                            boolean parent) throws IllegalAccessException,
                                                                   InvocationTargetException {
-        Object fieldData = null;
+        Object fieldData;
         if (useGetterAndSetter) fieldData = useGetterIfExists(attributeProxy, object);
         else fieldData = attributeProxy.getValue(object);
-        record.setString(fieldData != null ? fieldData.toString() : null, index);
+        record.setString((fieldData != null) ? fieldData.toString() : null, index);
+        boolean isNull = fieldData == null;
```

Comment:
Give this a more meaningful name like `isFieldDataNull`. Also move this boolean assignment one line up and use it in the ternary operator.

Reply:
addressed.